guava/src/com/google/common/escape/Escapers.java

```
@@ -211,14 +215,19 @@ public static String computeReplacement(CharEscaper escaper, char c) {
    * @param cp the Unicode code point to escape if necessary
    * @return the replacement string, or {@code null} if no escaping was needed
    */
-  public static String computeReplacement(UnicodeEscaper escaper, int cp) {
+  public static String computeReplacement(UnicodeEscaper escaper, @NonNegative int cp) {
     return stringOrNull(escaper.escape(cp));
   }
 
   private static String stringOrNull(char[] in) {
     return (in == null) ? null : new String(in);
   }
 
+  @SuppressWarnings(value = {"assignment.type.incompatible",//checker was not able to parse
```

Similar advice here about being specific about the error being suppressed.